src/test/controllers/users.test.js

```diff
@@ -90,3 +91,21 @@ describe('User Registration test', () => {
       });
   });
 });
+
+
+describe('User controller unit test', () => {
+  it('return error for invalid request object', async () => {
+    // creating an empty req body to generate error
+    const req = {};
+
+    // mocking res.status().json
+    const res = {
+      status: () => ({ json: jsonVal => JSON.stringify(jsonVal) }),
+    };
+    // call controller with mocked req,res objects
+    let returnValue = await userController(req, res);
+    returnValue = JSON.parse(returnValue);
+    chai.expect(returnValue.status).equal(500);
+    chai.expect(returnValue).to.have.property('message');
+  });
+});
```

Comment:
well done @Oliver-ke , please what does this test do??

Reply:
Thanks @oyewoas , While completing the task some section of the code in users controller which was the catch block was not tested so code climate was erroring, I had to write a test for that block. Here I'm mocking a request that should throw an error and then I'm using chai expect to see if an error was truly return

Reply:
ok @Oliver-ke